basic-introduction-html-css/html-semantics/README.md

```diff
@@ -0,0 +1,71 @@
+# Script
+
+In this video, we will take a look at the semantics of HTML. In programming, semantics refers to the meaning conveyed.
+
+> Action: highlight the HTML code that we wrote previously.
+
+We have used the `<img>` tag to display an image. The `<img>` tag has the semantic meaning of including a picture on our webpage. Similarly, we used `<h1>` tag to display a heading. The `<h1>` element gives the text it wraps a semantic role of top-level heading on the web page.
+
+> Action: write a `span` with inline CSS styling and make it look like a heading
+
+We can make a text wrapped by `<span>` look like a heading, but it doesn't have any semantic value.
+
+Using the correct element for its intended usage has its benefits.
+
+An adequately structured web page can help a search engine extract essential keywords. It can act as a signpost for screen readers to help navigate a page for visually impaired users.
```

Comment:
can help a search engine (such as: Google, Yahoo or Bing),  to extract essential keywords.